bot.js

```diff
@@ -29,85 +29,87 @@ client.on('ready', () => {
 
 // Listen for incoming messages from users
 client.on('message', async message => {
+    if (message.from === `${process.env.PHONE_NUMBER_TO_CONTACT}@c.us`) {
```

Comment:
One issue with that is if the bot is not for testing purposes it'll doesn't work on any other chat.
Can you add if `PHONE_NUMBER_TO_CONTACT` is defined in the `.env` file so we can check the number, otherwise just let him run

Reply:
The issue has been resolved. Ready for review. Thank you!